Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json

```
@@ -45,13 +45,13 @@
     "defaultName": "BlobTriggerCSharp",
     "postActions": [
         {
-            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Storage Nuget package",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs Nuget package",
             "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
             "ContinueOnError": "true",
             "ManualInstructions": [],
             "args": {
                 "referenceType": "package",
-            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs", "version": "5.1.2",
+            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs", "version": "6.0.0",
```

I think you might have accidentally undone the changes here:
```suggestion
                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs",
                "version": "6.0.0",
```